ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_virtual_topologies.schema.yml

```diff
@@ -0,0 +1,112 @@
+          id:
+            type: int
+            description: ID of the control plane AVT in the default VRF.
+            default: 254
+      policies:
+        type: list
+        description: |-
+          List of virtual toplogies policies.
+
+          For AutoVPN, each item in the list creates:
+            * one policy with:
+                * one `match` entry per `application_virtual_topologies` item
+                  they are indexed using `10 * <list_index>` where `list_index` starts at `1`.
+                * one `default-match`
+            * one load-balance policy per `application_virtual_topologies` and one for the `default_virtual_topology`.
+            * if the policy is associated with the default VRF, a special control-plane rule is injected
+              in the policy with index `1` referring to a control-plane load-balance policy as defined under
+              `control_plane_virtual_topology`.
+
+
+          For CV Pathfinder, each item in the list creates:
+            * one policy with:
+                * one `match` entry per `application_virtual_topologies` item ordered as in the model.
+                * one last match entry for the `default` application-profile using `default_virtual_topology` information.
+            * one profile per `application_virtual_topologies` item.
+            * one profile for the `default_virtual_topology`..
+            * one load-balance policy per `application_virtual_topologies`.
+            * one load_balance policy for the `default_virtual_topology`.
+            * if the policy is associated with the default VRF, a special control-plane profile is configured
+              and injected first in the policy assigned to the `default` VRF. This profile points to a
+              control-plane load-balance policy as defined under `control_plane_virtual_topology`.
+
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Name of the AVT policy.
+            application_virtual_topologies:
+              type: list
+              description: List of application specific virtual topologies.
+              primary_key: application_profile
+              items:
+                type: dict
+                $ref: "eos_designs#/$defs/application_virtual_topologies"
+                keys:
+                  application_profile:
+                    type: str
+                    description: The application profile to use for this virtual topology. It must be a defined `application_profile`.
+                  name:
+                    type: str
+                    description: Optional name, if not set `<policy_name>_<application_profile>` is used.
+                  id:
+                    type: int
+                    description: ID of the AVT in each VRFs. ID must be unique across all virtual topologies.
+            default_virtual_topology:
```

Comment:
Find out if default is always required. And if so we should have a default id and remove that from allowed range. (1 or 253 for default?)

Reply:
so it is not always required but we will reserve id 1 for it still and allow a drop boolean